core/web-assets/src/main/assets/js/apps/onms-classifications/index.js

```diff
@@ -125,14 +126,17 @@ const confirmTopoverTemplate = require('./views/modals/popover.html');
             };
 
             $scope.refreshTabs = function(navigateToFirstGroup) {
-                return ClassificationGroupService.query({}, function(response) {
+                return ClassificationGroupService.query({
+                    limit: 20,
```

Comment:
What happens if/when there are more than 20 groups? (We can solve this separately if it's non-trivial.)

Reply:
hm good point. I am not sure if there is a use case where we would have over 20 groups but maybe we just want to remove that limit?

Reply:
I put it to a 1000 :-)